pallets/subtensor/src/coinbase/run_coinbase.rs

```diff
@@ -131,6 +131,11 @@ impl<T: Config> Pallet<T> {
                     log::debug!("Accumulated emissions on hotkey {:?} for netuid {:?}: mining {:?}, validator {:?}", hotkey, *netuid, mining_emission, validator_emission);
                 }
             } else {
+                // No epoch, increase blocks since last step and continue
+                Self::set_blocks_since_last_step(
```

Comment:
It is ok to increase 1 block here. I just wondering why not record the block of last step. then blocks_since_last_step can be get by (current block - last step block). instead of plus 1 every time.

Reply:
The intention here is to reimplement logic that existed before the refactor and was unintentionally removed.